lib/ruby_lsp/test_reporters/minitest_reporter.rb

```diff
@@ -55,6 +55,8 @@ def reporters
   end
 
   class MinitestReporter < Minitest::AbstractReporter
+    NO_LOCATION = [nil, nil, nil].freeze
```

Comment:
Do we need this constant? Can we instead return `nil` directly when we cannot reconcile a test?